lib/git/base.rb

```diff
@@ -328,7 +328,14 @@ def push(remote = 'origin', branch = 'master', opts = {})
 
       self.lib.push(remote, branch, opts)
     end
-    
+
+    # aborts the current conflict resolution process, and try to reconstruct the pre-merge state.
+    # rauses an error if there are no umerged files
+    def abort_merge
+      raise "nothing to abort - no unmerged files" if self.lib.unmerged.count == 0
```

Comment:
you can use the `has_unmerged_files?` method instead of `self.lib.unmerged.count == 0`